biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -0,0 +1,65 @@
+package org.reactome.release;
+
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class ModelsTSVParserTester {
+    @Test
+    public void emptyMapFromEmptyFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("");
+        assertTrue("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromUnknownFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("filedoesnotexist.tsv");
+        assertTrue("Unknown file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromImproperFile() {
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/improper_models2pathways.tsv");
+        assertTrue("Improperly formatted file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void populatedMapFromPartiallyCorrectFile() {
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/partiallycorrect_models2pathways.tsv");
```

Comment:
Similar to: https://github.com/reactome/data-release-pipeline/pull/96/files#r333326084